neqo-crypto/src/lib.rs

```diff
@@ -167,6 +167,9 @@ pub fn init() -> Res<()> {
 ///
 /// If NSS cannot be initialized.
 pub fn init_db<P: Into<PathBuf>>(dir: P) -> Res<()> {
+    // Allow overriding the NSS database path with an environment variable.
+    let dir = env::var("NSS_DB_PATH")
+        .unwrap_or(dir.into().to_str().ok_or(Error::InternalError)?.to_string());
```

Comment:
This is not used by Firefox, right?

I just want to make sure that third parties like some Antivirus vendors don't have yet another way to mess with our stack.

Reply:
In Firefox, it's only used for tests: https://searchfox.org/mozilla-central/search?q=init_db%28&path=&case=false&regexp=false